enhancements/service-catalog/service-catalog-and-broker-removal.md

```diff
@@ -0,0 +1,266 @@
+remove them from productization. They will no longer be installed in a new
+cluster.
+
+## Motivation
+
+This is the next step in the removal of the Service Catalog and Brokers.
+
+* Deprecated APIs in OpenShift 4.1 [release
+  notes](https://docs.openshift.com/container-platform/4.1/release_notes/ocp-4-1-release-notes.html).
+* In OpenShift 4.2, a more detailed announcement of their deprecation was
+  released: [4.2 release notes](https://docs.openshift.com/container-platform/4.2/release_notes/ocp-4-2-release-notes.html#ocp-41-deprecated-features)
+* [Support for templates](https://docs.openshift.com/container-platform/4.2/release_notes/ocp-4-2-release-notes.html#ocp-4-2-general-web-console-updates) without using the Template Service Broker and Service
+  Catalog added to OpenShift 4.2.
+* Alerts were added to the Service Catalog operators:
+  * https://github.com/openshift/cluster-svcat-apiserver-operator/blob/master/manifests/0000_90_cluster-svcat-apiserver-operator_02-operator-servicemonitor.yaml#L28-L45
+  * https://github.com/openshift/cluster-svcat-controller-manager-operator/blob/master/manifests/0000_90_cluster-svcat-controller-manager-operator_02_servicemonitor.yaml#L27-L44
+* Alerts were added to the Ansible Service Broker operator:
+  * https://github.com/openshift/ansible-service-broker/blob/master/operator/roles/ansible-service-broker/templates/broker.service-monitor.yaml.j2
+* Alerts were added to the Template Service Broker operator:
+  * https://github.com/openshift/template-service-broker-operator/blob/master/roles/template-service-broker/templates/tsb.prometheus-alert.yaml
+
+### Goals
+
+1. Implement a mechanism that ensures the Service Catalog and Service Brokers
+   are not present after an upgrade to OpenShift 4.4. (ENG)
+1. Add a new operator to remove brokers to the OpenShift 4.4 release payload (ENG/ART)
+1. Remove the Service Catalog from the OpenShift 4.4 release payload (ART)
+   * Will have to keep the Service Catalog operators
+1. Remove the Template Service Broker from productization (ART)
+1. Remove the Ansible Service Broker from productization (ART)
+1. Remove the Template Service Broker Operator from productization (ART)
+1. Remove the Ansible Service Broker Operator from productization (ART)
+1. Remove the Service Catalog from productization (ART)
+
+### Non-Goals
+
+N/A
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+Implement a mechanism that ensures the Service Catalog is not present after
+an upgrade to OpenShift 4.4.
+
+Replace the manifests of the existing cluster-svcat-apiserver-operator and
+cluster-svcat-controller-manager-operator, with a [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/) definition.
+This might be accomplished with just one of the operators, that would need
+a little more investigation.
+
+Replace the operator's code with a new Job executable that will perform the
```

Comment:
We need to make sure that this orphan's everything. I thought we talked about removing these resources be removing the API server and *not* actually deleting these resources, correct?

Reply:
That was done in either 4.2 or 4.3, but we remove the owner references and orphan things. So we're good there.